packages/trpc/server/routers/loggedInViewer/appCredentialsByType.handler.ts

```diff
@@ -9,7 +11,34 @@ type AppCredentialsByTypeOptions = {
   input: TAppCredentialsByTypeInputSchema;
 };
 
+/** Used for grabbing credentials on specific app pages */
 export const appCredentialsByTypeHandler = async ({ ctx, input }: AppCredentialsByTypeOptions) => {
   const { user } = ctx;
-  return user.credentials.filter((app) => app.type == input.appType).map((credential) => credential.id);
+  const userAdminTeams = await getUserAdminTeams({ userId: ctx.user.id, getUserInfo: true });
+
+  const teamIds = userAdminTeams.reduce((teamIds, team) => {
```

Comment:
I'm getting a weird type error here. The type check thinks that `userAdminTeams` is type never but locally, it's being typed correctly.

![image](https://github.com/calcom/cal.com/assets/65426560/1f15e3c5-c6a2-487f-8ffb-20efdbf201a0)